.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```

Comment:
Any reason why this max length is long?

Reply:
That's what I typically use. I find that with today's screens 80 is too limiting.

Reply:
Gotcha! Okay. That sounds good.